R/rdeephaven/R/agg_ops_wrapper.R

```diff
@@ -1,6 +1,134 @@
+# An AggOp represents an aggregation operator that can be passed to agg_by() or agg_all_by(). This is the return type
+# of all of the agg functions. It is a wrapper around an Rcpp_INTERNAL_AggOp, which itself is a wrapper around a
+# C++ AggregateWrapper, which is finally a wrapper around a C++ Aggregate. See rdeephaven/src/client.cpp for details.
+# Note that AggOps should not be instantiated directly by user code, but rather by provided agg functions.
+
+
+#' @name
+#' AggBy
+#' @title
+#' Aggregations in Deephaven
+#' @md
+#' @usage NULL
+#' @format NULL
+#' @docType class
+#'
+#' @description
+#' Table aggregations are a quintessential feature of Deephaven. You can apply as many aggregations as
+#' needed to static tables _or_ streaming tables, and if the parent tables are streaming, the resulting aggregated
+#' tables will update alongside their parent tables. It is also very easy to perform _grouped_ aggregations, which
+#' allow you to aggregate tables on a per-group basis.
+#'
+#' @section
+#' Apply aggregations to a table:
+#' There are two methods for performing aggregations on a table, `agg_by()` and `agg_all_by()`. `agg_by()` allows you to
+#' perform many aggregations on specified columns, while `agg_all_by()` allows you to perform a single aggregation to
+#' every non-grouping column in the table. Both methods have an optional `by` parameter that is used to specify grouping columns.
+#' Here are some details on each method:
+#'
+#' - `TableHandle$agg_by(aggs, by)`: Creates a new table containing grouping columns and grouped data.
+#'   The resulting grouped data is defined by the aggregation(s) specified.
+#' - `TableHandle$agg_all_by(agg, by)`: Creates a new table containing grouping columns and grouped data.
+#'   The resulting grouped data is defined by the aggregation specified. This method applies the aggregation to all
+#'   non-grouping columns of the table, so it can only accept one aggregation at a time.
+#'
+#' The `agg_by()` and `agg_all_by()` methods themselves do not know anything about the columns on which you want to
+#' perform aggregations. Rather, the desired columns are passed to individual `agg` functions, enabling you to apply
+#' various kinds of aggregations to different columns or groups of columns as needed.
```

Comment:
I'm not sure if `agg_all_by` operators accept column args.  I think they don't, but this wording sounds like they do.